src/Features/ScreenSpaceGI.cpp

```diff
@@ -530,13 +530,11 @@ void ScreenSpaceGI::UpdateSB()
 	auto viewport = RE::BSGraphics::State::GetSingleton();
 	auto& state = State::GetSingleton()->shadowState;
 
-	uint resolution[2] = {
-		(uint)(State::GetSingleton()->screenWidth * viewport->GetRuntimeData().dynamicResolutionCurrentWidthScale),
-		(uint)(State::GetSingleton()->screenHeight * viewport->GetRuntimeData().dynamicResolutionCurrentWidthScale)
-	};
-	uint halfRes[2] = { (resolution[0] + 1) >> 1, (resolution[1] + 1) >> 1 };
-
-	float2 res = settings.HalfRes ? float2{ (float)halfRes[0], (float)halfRes[1] } : float2{ (float)resolution[0], (float)resolution[1] };
+	float2 res = { (float)texRadiance->desc.Width, (float)texRadiance->desc.Height };
+	float2 dynres = res * viewport->GetRuntimeData().dynamicResolutionCurrentWidthScale;
```

Comment:
Usually we use dynamicResolutionCurrentHeightScale for y. Are they always the same?